### PR TITLE
Tweak layout of quo/author component to use flex-baseline

### PR DESCRIPTION
**Pull request by seanstrom**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary

* This PR follows the work from  #19468
* This PR is based on the designs found here: [Figma Designs](https://www.figma.com/file/jGs1P4RU4R1NRrY0xXmHC1/Profile-interactions?type=design&node-id=1368-136256&mode=design&t=Du3kOGa2eVFAALp3-4)
* This PR attempts to change the quo/author component to match the alignment of elements in the Figma designs.
* This PR could `skip-manual-qa` since it mainly tweaks the text alignment for the `quo/author` component.

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats
- Home contacts list
- Home recent chats list

### Steps to test

WIP

##### Home recent chats list
Screenshot 2024-04-03 at 09 43 33

##### Home contacts list
Screenshot 2024-04-03 at 09 44 52

##### 1-1 chat
Screenshot 2024-04-03 at 09 46 08

##### Home recent chats list
Screenshot 2024-04-03 at 09 44 07

##### Home contacts list
Screenshot 2024-04-03 at 09 45 01

##### 1-1 chat
Screenshot 2024-04-03 at 09 46 30

##### Home recent chats list
Screenshot 2024-04-03 at 10 05 25

##### Home contacts list
Screenshot 2024-04-03 at 10 06 01

##### 1-1 chat
Screenshot 2024-04-03 at 10 07 37

##### Home recent chats list
Screenshot 2024-04-03 at 10 05 40

##### Home contacts list
Screenshot 2024-04-03 at 10 06 46

##### 1-1 chat
Screenshot 2024-04-03 at 10 07 49

status: ready

### CODE REVIEW

File: src/quo/components/messages/author/style.cljs [hunk 1]
```diff
@@ -38,7 +41,10 @@
 (defn icon-container
   [is-first?]
   {:margin-left   (if is-first? 4 2)
-   :margin-bottom 2})
+   :margin-bottom 0
```
Comment (ibrkhalil) [inline src/quo/components/messages/author/style.cljs]: Do we need this zero value?

**Review by ibrkhalil — APPROVED** [review]
Nice work :)

**Review by mohsen-ghafouri — APPROVED** [review]
Looks great

File: src/quo/components/messages/author/style.cljs (hunk 1 above)
Comment (seanstrom) [reply src/quo/components/messages/author/style.cljs]: Nice catch! I'll remove this 👍

**Review by ulisesmac — APPROVED** [review]
@seanstrom  Thanks for this PR!

**Comment by seanstrom** [comment]
@status-im/mobile-qa can you review these E2E results please? 🙏

**Comment by yevh-berdnyk** [comment]
> @status-im/mobile-qa can you review these E2E results please? 🙏

hi @seanstrom, e2e failures are not related the PR

**Comment by seanstrom** [comment]
@Francesca-G can you please design review these UI changes? 🙏 
I'm attempting to get this as close as possible to the designs, but I think I've been staring at this too long 😸

**Review by Francesca-G — APPROVED** [review]
Looks good to me ✨

**Pull request merged**